vowpalwabbit/gd.cc

```diff
@@ -17,8 +17,12 @@ license as described in the file LICENSE.
 #define __SSE2__
 #endif
 
-#if defined(__ARM_NEON__)
+#if defined(__ARM_NEON__) || defined(__ARM_NEON)
```

Comment:
Nitpick: do you know if we need both? According to https://developer.arm.com/documentation/ihi0053/d/?lang=en only `defined(__ARM_NEON)` should be needed (`defined(__ARM_NEON__)` apparently is the legacy way).

Reply:
only `__ARM_NEON` is activated on a graviton instance. I will remove the other flag.